docs/admin/install/vm-install/rhel.md

```diff
@@ -102,6 +102,33 @@ sudo /opt/jans/jans-cli/jans_cli_tui.py
 
 Full TUI documentation can be found [here](../../config-guide/config-tools/jans-tui/README.md)
 
+If you have selected casa during installation you can access casa using url``` https://<host>/jans-casa ```
+
+## Let's Encrypt
```

Comment:
This section primarily has two parts:
1) Instructions for getting certificate (from Let's encrypt)
2) Instructions for configuring certificates in Jans

I believe this document should only contain the `2` as it pertains to Jans. Let's encrypt being one of many ways of getting certificates, it shouldn't be part of install instructions. May be it should be moved to `how-to` docs and give a link here.

Reply:
created how-to doc https://github.com/JanssenProject/jans/pull/6323
and kept only jans related instruction ie  point 2